includes/class-used-css.php

```
@@ -0,0 +1,1206 @@
+			if ( ':root' === $simple || ':host' === $simple ) {
+				return true;
+			}
+
+			if ( false !== strpos( $simple, ':' ) ) {
+				// Strip pseudo-classes/elements using a depth-tracking parser
+				// that handles nested parentheses like :not(.a:not(.b)).
+				$stripped = '';
+				$depth    = 0;
+				$len      = strlen( $simple );
+				for ( $i = 0; $i < $len; ++$i ) {
+					$ch = $simple[ $i ];
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ch. Configured minimum length is 3. (undefined)

(ShortVariable)